sdk/src/main/java/com/qonversion/android/sdk/internal/QRemoteConfigManager.kt

```diff
@@ -71,6 +72,16 @@ internal class QRemoteConfigManager @Inject constructor(
         remoteConfigService.detachUserFromExperiment(experimentId, internalConfig.uid, callback)
     }
 
+    fun attachUserToRemoteConfiguration(remoteConfigurationId: String, callback: QonversionRemoteConfigurationAttachCallback) {
+        currentRemoteConfig = null
+        remoteConfigService.attachUserToRemoteConfiguration(remoteConfigurationId, internalConfig.uid, callback)
```

Comment:
As I see, we use `internalConfig` in this class just to get uid. If so, let's replace `internalConfig` in the constructor with the `UidProvider` and the variable name. It would be enough. Nothing else needs changes.